File: python/27_count_unique_chars.py
"""Contar caracteres únicos — Fácil · Strings

Devuelve la cantidad de caracteres distintos, distinguiendo mayúsculas y
minúsculas. Los espacios también cuentan.

Fuente: https://coding-challenges.dev/problems/python-typescript-contar-caracteres-unicos
"""


def count_unique_chars(text: str) -> int:
    return len(set(text))

File: typescript/27-count-unique-chars.ts
// Contar caracteres únicos — Fácil · Strings
// Cuenta caracteres distintos, distinguiendo mayúsculas y minúsculas.
// Fuente: https://coding-challenges.dev/problems/typescript-contar-caracteres-unicos

export function countUniqueChars(text: string): number {
  return new Set(text).size;
}
